Packs/Salesforce/Integrations/SalesforceV2/SalesforceV2.py

```diff
@@ -0,0 +1,1041 @@
+                # retrieve owner info
+                cases[index]['OwnerDetails'] = client.getObject(item.get('OwnerId'))
+                cases[index]['LastModifiedDate'] = parse(
+                    item.get('LastModifiedDate')).isoformat().split("+")[0].split(".")[0] + "Z"
+                cases[index]['CreatedDate'] = parse(item.get('CreatedDate')).isoformat().split("+")[0].split(".")[0] + "Z"
+                incidents.append({
+                    "name": f"{item.get('Id')} {item.get('Subject')}",
+                    "details": item.get('Description'),
+                    "owner": cases[index]['OwnerDetails'].get("Email"),
+                    "occurred": parse(item.get('CreatedDate')).isoformat().split("+")[0].split(".")[0] + "Z",
+                    "rawJSON": json.dumps(item),
+                    'mirror_direction': MIRROR_DIRECTION.get(params.get('mirror_direction')),
+                    'mirror_tags': [params.get('comment_tag'), demisto.params().get('file_tag')],
+                    'mirror_instance': demisto.integrationInstance()
+                })
+
+    else:
+        # Fetch comment replies
+
+        properties += ['Id', 'CommentBody', 'CreatedDate']
+        condition = f"CreatedDate> {lastRun.get('last_case_time')} {demisto.params().get('condition','')} ORDER BY CreatedDate DESC LIMIT 10"
+        replies = client.queryObjects(list(set(properties)), "FeedComment", condition).get("records")
+
+        if len(replies) > 0:
+
+            lastRun['last_case_time'] = parse(replies[0].get("CreatedDate")).isoformat().split("+")[0].split(".")[0] + "Z"
+            demisto.setLastRun(lastRun)
+
+            for reply in replies:
+                # Get reply details
+                replyDetails = client.sendRequestInSession('GET', f'chatter/comments/{reply.get("Id")}')
+                feedElement = replyDetails.get("feedElement")
+                parentID = feedElement.get("id")
+
+                # get parent details
+                parentDetails = client.getObject(f'CaseFeed/{parentID}')
+                parentText = parentDetails
+
+                # if (parentText.indexOf('DemistoID') !== -1) {
+                messageSegments = replyDetails.get("body").get("messageSegments")
+
+                for ms in messageSegments:
+                    if ms.get('type') == 'Text':
+                        # Found the relevant comment (there's only one), so we return it
+                        incidents.append({
+                            "name": f"{parentText['attributes']['type']} {parentText['Id']}",
+                            "details": ms.get("text"),
+                            "occurred": parse(parentText.get('CreatedDate')).isoformat().split("+")[0].split(".")[0] + "Z",
+                            "rawJSON": json.dumps(parentText)
+                        })
+
+    return incidents
+
+
+def test_module(client):
+    token = client.getNewToken()
+    if token.get("access_token"):
+        return 'ok'
```

Comment:
The test module should test the fetch_incidents command too.

Reply:
since when are we testing the fetch_incidents? what if there are no cases? please explain where this is documented in the developer articles @Noy-Maimon

Reply:
This comment is not relevant anymore because the Pack's support level has changed to - 'community'.